libvast/vast/fbs/partition.fbs

```diff
@@ -18,22 +18,38 @@ table QualifiedValueIndex {
   index: ValueIndex;
 }
 
+table TypeIds {
```

Comment:
Could you provide documentation about this type? What are its semantics?

```suggestion
table EventIDs {
```

A "type ID" to me is a single identifier for a type, but this is an event ID.